backend/box/api/types.go

```diff
@@ -72,20 +72,31 @@ const (
 	ItemStatusDeleted = "deleted"
 )
 
+// ItemMini is a subset of the elements in a full Item returned by some API calls
+type ItemMini struct {
+	Type       string `json:"type"`
+	ID         string `json:"id"`
+	SequenceID string `json:"sequence_id"`
```

Comment:
Perhaps this should be `int64`? Its defined to be numeric isn't it? You'd decode it like this to tell Go it is an int wrapped in a string.

```go
type ItemMini struct {
    // ...
	SequenceID string `json:"sequence_id,string"`
    // ...
}
```

That would then remove all the ParseInt's from the code below.

Reply:
Thanks, didn't realize this was possible. Done.